README.md

```diff
@@ -0,0 +1,63 @@
+# IronSSH - End-to-end secure file transfer
+
+This project is a fork of the *openssh/openssh-portable* project. While most of the project is unchanged, specific additions have been made to create new executables that provide end-to-end security when transferring files to remote servers.
+
+While *sftp* and *scp* use *ssh* to keep files secure while they are being transferred over the network, once those files hit the remote server, they are no longer protected. The **ironsftp** executable provides additional security. When you put a file on the server using **ironsftp**, the file is encrypted before it is uploaded, and it stays that way on the server. When you get a file from the server, it is downloaded then decrypted. So the file remains secure until it is at the place you want to use it - on your local machine.
+
+The extension '.iron' is used to denote secured files. If you run **ironsftp** and put 'foo.c' on the server, it will be written remotely as 'foo.c.iron'. When you get 'foo.c', **ironsftp** will look for 'foo.c', but if it doesn't find that file, it will try to download 'foo.c.iron'. If that is successful, it will decrypt that file and write 'foo.c' on your local machine. The process operates the same as *sftp*, but your files are protected on the remote server.
```

Comment:
Also, maybe spell out that fetching `foo.c` getting `foo.c.iron` is just a convenience and the full filename can be specified for completeness or to resolve a conflict when both `foo.c` and `foo.c.iron` exist on the server.